react/components/EditorContainer/Sidebar/ConfigurationList/index.tsx

```diff
@@ -345,6 +350,54 @@ class ConfigurationList extends Component<Props, State> {
     }
   }
 
+  private handleContentDelete: MutationFn<
+    { deleteContent: string },
+    DeleteContentVariables
+  > = async options => {
+    const { editor, iframeRuntime } = this.props
+    editor.setIsNavigating(true)
+    try {
+      await this.props.deleteContent({
+        ...options,
+        update: (store, { data }) => {
+          const variables = {
+            pageContext: iframeRuntime.route.pageContext,
+            template: iframeRuntime.pages[iframeRuntime.page].blockId,
+            treePath: editor.editTreePath,
+          }
+
+          const { listContent } = store.readQuery<{
+            listContent: ExtensionConfiguration[]
+          }>({ query: ListContent, variables }) || { listContent: [] }
+          const newConfigurations = {
+            listContent: listContent.filter(
+              ({ contentId }) => contentId !== data!.deleteContent
+            ),
+          }
+          store.writeQuery({
+            data: newConfigurations,
+            query: ListContent,
+            variables,
+          })
+        },
+      })
+
+      editor.setIsNavigating(false)
+      this.props.showToast({
+        horizontalPosition: 'right',
+        message: 'Content deleted.',
+      })
+    } catch (e) {
+      editor.setIsNavigating(false)
+      this.props.showToast({
+        horizontalPosition: 'right',
+        message: 'Something went wrong. Please try again.',
```

Comment:
Missing i18n. I think we also need to do it for all the other Toast messages.

Reply:
Let's do this in another story (so we resolve all pending issues)!